OJP/OJP_PlaceSupport.xsd

```diff
@@ -460,6 +460,11 @@
 					<xs:documentation>Public name of the place.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="AllowedSystemId" type="xs:normalizedString" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Used in distributed environments. e.g. EU-Spirit. If none is given, the place information request refers to all known systems (in EU-Spirit "passive servers"). If at least one is given, the place information request refers only to the given systems (in EU-Spirit "passive servers"). In EU-Spirit the system IDs were previously called "provider code". See https://eu-spirit.eu/</xs:documentation>
```

Comment:
The text still says "If at least one is given" hinting there could be multiple but the unbounded has been removed recently. I don't know whether allowing only one or multiple `AllowedSystemId` elements is correct but the text should be accordingly

Reply:
From EU-Spirit's point of view "If at least one is given" is correct. Either the active system defines a list of allowed passive servers to be involved or leaves the list empty which means all systems may be involved.

Reply:
updated